template/README.md

```diff
@@ -15,7 +15,7 @@
 
 Hey there 👋,
 
-I create blog posts and open source packages mainly about python and data engineering.  I have a passion for learning and sharing my knowledge with others as public as possible.  You can see a full list of what I am up to on [waylonwalker.com](waylonwalker.com).  If you found value in something I have created, please feel free to send a [tip](https://www.buymeacoffee.com/bBdtMQO), give me a shout out [@_waylonwalker](https://twitter.com/_waylonwalker), give some ♥ on [DEV.to/waylonwalker](https://dev.to/waylonwalker), or sign up for my newsletter  at waylonwalker.com/newsletter  I would love to see you join.
```

Comment:
i think https:// -{HERE} is necessary in line -- You can see a full list of what I am up to on [waylonwalker.com] ( HERE waylonwalker.com).  Its showing web page not found please make it correct

Reply:
yes the `https://` with the website to make the hyperlink.